schema/property.go

```diff
@@ -28,14 +31,17 @@ type Property struct {
 	SQLType                string
 	OnDeleteCascade        bool
 	Default                interface{}
+	DefaultMask            interface{}
 	Indexed                bool
 }
 
 //PropertyMap is a map of Property
 type PropertyMap map[string]Property
 
+const ItemPropertyID = "[]"
+
 //NewProperty is a constructor for Property type
-func NewProperty(id, title, description, typeID, format, relation, relationColumn, relationProperty, sqlType string, unique, nullable, onDeleteCascade bool, properties map[string]interface{}, defaultValue interface{}, indexed bool) Property {
+func NewProperty(id, title, description, typeID, format, relation, relationColumn, relationProperty, sqlType string, unique, nullable, onDeleteCascade bool, properties []Property, items *Property, defaultValue interface{}, indexed bool) Property {
```

Comment:
16 parameters is 'bit' too much, especially that there are often filled with `nil`. Please consider removing all but this necessary (like id, typeID) and some builder instead

Reply:
Removed in favor of the `PropertyBuilder`